src/categories.py

```diff
@@ -9,7 +9,7 @@
 CELLULAR_COMPONENT = 'biolink:CellularComponent'
 CHEMICAL_ENTITY = 'biolink:ChemicalEntity'
 CHEMICAL_MIXTURE = 'biolink:ChemicalMixture'
-COMPLEX_CHEMICAL_MIXTURE = 'biolink:ComplexMolecularMixture'
+COMPLEX_MOLECULAR_MIXTURE = 'biolink:ComplexMolecularMixture'
```

Comment:
This is admittedly not exactly the envisioned use case but the idea of these consts was so that when a 1:1 change occurred here you could just change the string and save yourself many edits throughout the code.

Reply:
Yes, I know! But I took advantage of the centralized constants to bulk-rename one that had fallen out of sync with the corresponding Biolink type. If it had been a large change I would have proposed it in its own PR, but since it was only used in a handful of places and was related to the overall PR changes, I decided to incorporate it in here. For future changes, I'll update the string first and only update the constant name once it's pretty clear Biolink isn't going to change it again any time soon.